bluebottle/initiatives/tests/test_api.py

```diff
@@ -1552,3 +1553,66 @@ def test_submitted_initiatives_dont_show_on_map(self):
 
         data = response.json()
         self.assertTrue(len(data), 5)
+
+    def test_locations(self):
+        InitiativeFactory(
+            status='approved',
+            place=GeolocationFactory(position=Point(23.6851594, 43.0579025))
+        )
+        InitiativeFactory(
+            status='approved',
+            place=GeolocationFactory(position=Point(27.6851594, 34.0579025))
+        )
+
+        response = self.client.get(
+            self.url
+        )
+        data = response.json()
+
+        lat = data[0]['position'][0]
+        long = data[0]['position'][1]
+        lat_2 = data[1]['position'][0]
+        long_2 = data[1]['position'][1]
+        self.assertTrue(lat in data[0]['position'])
+        self.assertTrue(long in data[0]['position'])
+        self.assertTrue(lat_2 in data[1]['position'])
+        self.assertTrue(long_2 in data[1]['position'])
+
+    def test_lat_long_omitted(self):
+        InitiativeFactory(
+            status='approved',
+            place=GeolocationFactory(position=Point(23.6851594, 0))
+        )
+        InitiativeFactory(
+            status='approved',
+            place=GeolocationFactory(position=Point(0, 34.0579025))
+        )
+        InitiativeFactory(
+            status='approved',
+            place=GeolocationFactory(position=Point(23.6851594, 34.0579025))
+        )
+
+        response = self.client.get(
+            self.url
+        )
+        data = response.json()
+
+        for i in range(len(data)):
+            if data[i]['position'][0] == 0.0:
+                del data[i]
+                break
```

Comment:
No idea what you are doing here...

Reply:
Removing the initiatives that have a lat that is 0

Reply:
But why? You want to test the result from the response, not changing the data here in a test.